### PR TITLE
create signup ui

### PR DESCRIPTION
**Pull request by jj904**

### CODE REVIEW

**Review by nanarboursal — APPROVED** [review]
sign up functionality remains in tact

File: angle-ui/src/Pages/SignUp/signup.css [hunk 1]
```diff
@@ -1,13 +1,32 @@
-.login-page {
-    background-color: #ffffff; /*Change later*/
-    width: 100%;
-    height: 92vh;
+
+.signup-page {
+    /*background-image: url("C:\Users\jgaeu\Angle\angle-ui\src\Images\background.png");
+    */
```
Comment (Aneesh-Pothuru) [inline angle-ui/src/Pages/SignUp/signup.css:4]: Please remove comments

File: angle-ui/src/Pages/SignUp/signup.css [hunk 2]
```diff
@@ -1,13 +1,32 @@
...
+    font-weight: bold;
     margin: 2%;
-}
\ No newline at end of file
+    width: 40%;
+    border:20px;
+   /*
+    background: -webkit-linear-gradient(to right, #ee7724, #d8363a, #dd3675, #b44593);
+    background: linear-gradient(to right, #ee7724, #d8363a, #dd3675, #b44593);
+    */
```
Comment (Aneesh-Pothuru) [inline angle-ui/src/Pages/SignUp/signup.css:31]: Please remove comments

**Comment by Aneesh-Pothuru** [comment]
Hey Grace,
This is fine to merge, it works on my machine. Please remove the comments if you can. 
Thanks,
Aneesh

**Review by Aneesh-Pothuru — CHANGES_REQUESTED** [review]
Hey Grace,
I actually ran this commit in parallel to Edgar's changes. This is actually breaking the user interface from the different styling logic you both used. Please go over this with each other and submit changes.

**Pull request merged**